src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/CreateVariantIngestFiles.java

```diff
@@ -288,8 +290,16 @@ public void onTraversalStart() {
         final GenomeLocParser genomeLocParser = new GenomeLocParser(seqDictionary);
         intervalArgumentGenomeLocSortedSet = GenomeLocSortedSet.createSetFromList(genomeLocParser, IntervalUtils.genomeLocsFromLocatables(genomeLocParser, intervalArgumentCollection.getIntervals(seqDictionary)));
 
+        if (gqStateToIgnore != null) {
```

Comment:
is this null check needed as `gqStateToIgnore` is initialized (to the questionable value of `GQStateEnum.SIXTY`) on line 60?

Reply:
Yes, I figured I'd future proof the code.